test/unit/specs/validators/email.spec.js

```diff
@@ -0,0 +1,48 @@
+import email from 'src/validators/email'
+
+describe('email validator', () => {
+  it('should not validate empty string', () => {
+    expect(email('')).to.be.false
```

Comment:
This actually should be true, otherwise you imply that the field is required. We have a separate validator for that.